### ISSUE TITLE
Regular weight: clean up outlines for numerals

### ISSUE THREAD

**Issue by eliheuer** [eliheuer/Libre-Plantin#4]
The numerals need some work.

**Comment by StefanPeev** [comment]
@eliheuer @davelab6 Yes, I know. I prepare a new version – with localization for Bulgarian Cyrillic, Serbian Cyrillic and Macedonian Cyrillic. In this version I'll try to arrange the numerals also. Any other suggestions?

**Comment by eliheuer** [comment]
I'm printing and reviewing proofs now. I'll add some more issues soon. The "Q" had some issues, but besides that really great work so far! :)

I'm using DrawBot[1] for proofs right now, but I'm not happy with that because it's not cross-platform. I'm thinking about remaking the proofs using ConTeXt[2]. 

[1]: http://www.drawbot.com/ 
[2]: http://wiki.contextgarden.net/Main_Page

**Comment by StefanPeev** [comment]
@eliheuer Well, I'm not familiar neither with DrayBot, nor with ConTeXt, so... it remains for you :) . Thanks for the positive evaluation of my work as a hole!

**Comment by eliheuer** [comment]
One more thing, If you make another pull request could you just work in the main .ufo here: https://github.com/eliheuer/Libre-Plantin/tree/master/sources/Libre-Plantin-Regular.ufo

That way it will be easy to merge changes. Thanks! Let me know if you have any questions. :)

**Comment by StefanPeev** [comment]
@eliheuer I see from the new picture that you have changed Q glyph. How could I know about your changes?

**Comment by eliheuer** [comment]
That's why working with git + UFO is the best way I know to collaborate on type design projects. 

In a UFO, each Glyph is its own .glif file. 

So, you could make a branch and change just the numerals and I could be in a branch working on just the "Q" glyph. Then, we could both make pull requests to master and our changes would merge without conflicts. 

This works best when we merge early and often. A pull request with 100+ glyphs changed can be difficult to review.
